### ISSUE TITLE
--help not working in benchmarking

### ISSUE THREAD

**Issue by georgebisbas** [devitocodes/devito#1128]
benchmark.py run/test/bench/run-jit-backdoor --help is broken:

---To reproduce:
`python benchmark.py run --help`

--Log:
```
python benchmarks/user/benchmark.py run --help
Traceback (most recent call last):
  File "benchmarks/user/benchmark.py", line 507, in <module>
    benchmark()
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 1135, in invoke
    sub_ctx = cmd.make_context(cmd_name, args, parent=ctx)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 641, in make_context
    self.parse_args(ctx, args)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 940, in parse_args
    value, args = param.handle_parse_result(ctx, opts, args)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 1477, in handle_parse_result
    self.callback, ctx, self, value)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 96, in invoke_param_callback
    return callback(ctx, param, value)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 860, in show_help
    echo(ctx.get_help(), color=ctx.color)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 516, in get_help
    return self.command.get_help(self)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 879, in get_help
    self.format_help(ctx, formatter)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 898, in format_help
    self.format_options(ctx, formatter)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 919, in format_options
    rv = param.get_help_record(ctx)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 1700, in get_help_record
    rv = [_write_opts(self.opts)]
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 1697, in _write_opts
    rv += ' ' + self.make_metavar()
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/core.py", line 1371, in make_metavar
    metavar = self.type.get_metavar(self)
  File "/home/gb4018/anaconda3/envs/devito/lib/python3.7/site-packages/click/types.py", line 149, in get_metavar
    return '[%s]' % '|'.join(self.choices)
TypeError: sequence item 3: expected str instance, tuple found
```

**Comment by georgebisbas** [comment]
Derives from autotuning and dle tuple configuration.

**Comment by FabioLuporini** [comment]
fixed by #1190
